### ISSUE TITLE
2.8.0 can't be extracted by tar on macOS 10.14 and older

### ISSUE THREAD

**Issue by jmroot** [libsdl-org/SDL_image#404]
`/usr/bin/tar` fails to extract the release tarball with this message:
```
tar: Ignoring malformed pax extended attribute
tar: Error exit delayed from previous errors.
```

Installing a recent libarchive and using its tar is an effective workaround, but it would be nice if it wasn't necessary. Extracting with GNU tar (on a different system in this example) sheds a little more light on what the problem is:
```
/usr/bin/gnutar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:kMDItemTextContentLanguage'
/usr/bin/gnutar: Ignoring unknown extended header keyword `SCHILY.xattr.com.apple.metadata:kMDItemTextContentLanguage'
/usr/bin/gnutar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:kMDItemKeyphraseVersion'
/usr/bin/gnutar: Ignoring unknown extended header keyword `SCHILY.xattr.com.apple.metadata:kMDItemKeyphraseVersion'
/usr/bin/gnutar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:kMDItemKeyphraseLabels'
/usr/bin/gnutar: Ignoring unknown extended header keyword `SCHILY.xattr.com.apple.metadata:kMDItemKeyphraseLabels'
/usr/bin/gnutar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:kMDItemKeyphraseConfidences'
/usr/bin/gnutar: Ignoring unknown extended header keyword `SCHILY.xattr.com.apple.metadata:kMDItemKeyphraseConfidences'
```

Downstream report: https://trac.macports.org/ticket/68881

**Comment by sezero** [comment]
The issue is reproducible on linux, too:

```
$ rpm -q tar
tar-1.23-15.el6_8.i686

$ tar xf SDL2_image-2.8.0.tar.gz
tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:kMDItemTextContentLanguage'
tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:kMDItemKeyphraseVersion'
tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:kMDItemKeyphraseLabels'
tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:kMDItemKeyphraseConfidences'
```

**Comment by slouken** [comment]
This is fixed for 2.8.1, thanks!
